Resources/Prototypes/Catalog/Fills/Lockers/heads.yml

```diff
@@ -25,6 +25,8 @@
         prob: 1
       - id: CigPackGreen
         prob: 0.50
+      - id: DoorRemoteCargo
+        prob: 1
```

Comment:
Isn't prob: 1 the default?

Reply:
Probably, I just copied what I saw other entries doing.